.github/workflows/CI.yml

```diff
@@ -15,26 +15,25 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1
+          - "1"  # Latest Releae
         os:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
         arch:
           - x64
+          - x86
+        exclude:
+          # Test 32-bit only on Linux
+          - os: macOS-latest
+            arch: x86
+          - os: windows-latest
+            arch: x86
         include:
-          # Add a 1.0 job just to make sure we still support it
-          - os: ubuntu-latest
-            version: 1.0.5
-            arch: x64
           # Add a 1.3 job because that's what Invenia actually uses
           - os: ubuntu-latest
             version: 1.3
             arch: x64
-          # Add a 32-bit job to ensure we don't have any 64-bit specific logic
-          - os: ubuntu-latest
-            version: 1
```

Comment:
I understand you're removing this 32-bit test in favour of using the `exclude`.

But do we not also want to keep testing against `1`? Currently we're only doing `1.3`? Or am I misreading this template? (All these `versions` are ambiguous and confusing)